### DISCUSSION TITLE
[remote_config] Select a specific FirebaseApp

### DISCUSSION THREAD

**Discussion by nthungdev** [firebase/flutterfire#5576] [Feature request] [answered]
My Android and iOS app is using multiple FirebaseApp. Unlike the Firestore and FirebaseAuth plugins, RemoteConfig doesn't have an api to select which FirebaseApp to use. I understand RemoteConfig must be initialize in the beginning, but it only initialize with the default FirebaseApp.

A potential syntax can be like below.
```dart
_remoteConfig = await RemoteConfig.initialize(name: <FirebaseApp name>);
```

**Comment by aidanm1999** [comment, marked as answer]
Hi @nthungdev I think this has now been added. If you go to the [FlutterFire docs](https://firebase.flutter.dev/docs/remote-config/usage/), you should be able to now add remote config for a specific app as follows: 

```dart
FirebaseApp secondaryApp = Firebase.app('SecondaryApp');
FirebaseRemoteConfig remoteConfig = FirebaseRemoteConfig.instanceFor(app: secondaryApp); 
```

Hope this helps!

**Reply by nthungdev** [reply]
Thank you very much for the update.
